### PR TITLE
Ex 73 add tuples to models

### PR DESCRIPTION
**Pull request by KellyInnovation**

### CODE REVIEW

File: lfucg-exactions/client/js/components/AccountLedgerForm.js [hunk 1]
```diff
@@ -101,7 +111,7 @@ class AccountLedgerForm extends React.Component {
                                 <fieldset>
                                     <div className="row">
                                         <div className="col-sm-6 form-group">
-                                            <label htmlFor="lot_id" className="form-label" id="lot_id">Lot</label>
+                                            <label htmlFor="lot_id" className="form-label" id="lot_id" aria-label="Required Lot">* Lot</label>
```
Comment (Wei-KeatKong) [inline lfucg-exactions/client/js/components/AccountLedgerForm.js]: you can use aria-required="true" to tell screen readers that this field is required.

File: lfucg-exactions/client/js/components/AccountLedgerForm.js [hunk 2]
```diff
@@ -165,30 +175,37 @@ class AccountLedgerForm extends React.Component {
                                     </div>
                                     <div className="row">
                                         <div className="col-sm-6">
-                                            <FormGroup label="Entry Date" id="entry_date">
+                                            <FormGroup label="* Entry Date" id="entry_date">
```
Comment (Wei-KeatKong) [inline lfucg-exactions/client/js/components/AccountLedgerForm.js]: no aria label here?

File: lfucg-exactions/client/js/components/AgreementForm.js [hunk 3]
```diff
@@ -120,10 +120,17 @@ class AgreementForm extends React.Component {
...
                                     <div className="row">
-                                        <div className="col-sm-6">
-                                            <FormGroup label="Agreement Type" id="agreement_type">
-                                                <input type="text" className="form-control" placeholder="Agreement Type" />
-                                            </FormGroup>
+                                        <div className="col-sm-6 form-group">
+                                            <label htmlFor="agreement_type" className="form-label" id="agreement_type" aria-label="Agreement Type">Agreement Type</label>
+                                            <select className="form-control" id="agreement_type" onChange={formChange('agreement_type')} >
+                                                {agreements.agreement_type ? (
+                                                    <option value="agreement_type" aria-label={`Agreement Type ${agreements.agreement_type_display}`}>{agreements.agreement_type_display}</option>
```
Comment (Wei-KeatKong) [inline lfucg-exactions/client/js/components/AgreementForm.js]: different styled quotes here?

File: lfucg-exactions/client/js/components/ProjectForm.js [hunk 4]
```diff
@@ -108,25 +116,104 @@ class ProjectForm extends React.Component {
...
+                                                ) : (
+                                                    <option value="choose_project_status" aria-label="Choose a Project Status">Choose a Project Status</option>
+                                                )}
+                                                <option value={['IN_PROGRESS', 'In Progress']}>In Progress</option>
+                                                <option value={['COMPLETE', 'Complete']}>Complete</option>
+                                            </select>
                                         </div>
                                         <div className="col-sm-6">
-                                            <FormGroup label="Status Date" id="status_date" >
+                                            <FormGroup label="* Status Date" id="status_date" >
```
Comment (Wei-KeatKong) [inline lfucg-exactions/client/js/components/ProjectForm.js]: aria-required

File: lfucg-exactions/client/js/components/AccountLedgerForm.js (hunk 2 above)
Comment (KellyInnovation) [inline lfucg-exactions/client/js/components/AccountLedgerForm.js]: The FormGroup component adds an aria-label

File: lfucg-exactions/client/js/components/AgreementForm.js (hunk 3 above)
Comment (KellyInnovation) [inline lfucg-exactions/client/js/components/AgreementForm.js]: Adding display on the end presents the display option from a tuple.  The different kinds of quotes allows for incorporation of data.

**Pull request merged**